apps/ui/lib/ui_web/live/profit_indicator_live.ex

```diff
@@ -93,32 +93,23 @@ defmodule UiWeb.ProfitIndicatorLive do
       %{
         :symbol => symbol,
         :type => :day,
-        :total => get_profit_base_currency_from_day(symbol)
+        :total => get_profit_base_currency(1, :day, symbol)
       },
       %{
         :symbol => symbol,
         :type => :week,
-        :total => get_profit_base_currency_from_week(symbol)
+        :total => get_profit_base_currency(1, :week, symbol)
       },
       %{
         :symbol => symbol,
-        :type => :all,
-        :total => get_profit_base_currency(symbol)
+        :type => :year,
```

Comment:
Not sure why you dropped here the full profit since start (`:all`)

Reply:
now `:all` back too